### ISSUE TITLE
No text is shown when starting lillygui.

### ISSUE THREAD

**Issue by wilsonk** [tauraamui/lilly#219]
This may be due to the recent changes for the splash screen? Perhaps we should just skip over the splash screen in the gui, for now (as suggested by tauraamui)?

**Comment by tauraamui** [comment]
Yes, I suspect the issue is the font used doesn't support the braille characters we're using to render the splash screen ASCII version of the Lilly logo.
